### PR TITLE
streamlined XSLT files

### PR DESCRIPTION
**Pull request by karindalziel**

### CODE REVIEW

File: lib/xslt/tei_to_html/lib/formatting.xsl [hunk 1]
```diff
@@ -1,51 +1,63 @@
...
+      <xsl:attribute name="rel">
+        <xsl:text>tooltip</xsl:text>
+      </xsl:attribute>
+      <xsl:attribute name="class">
+        <xsl:text>abbr</xsl:text>
+      </xsl:attribute>
+      <xsl:attribute name="title">
+        <xsl:apply-templates select="expan"/>​ </xsl:attribute>
+      <xsl:apply-templates select="abbr"/>
+    </a>​ </xsl:template>
```
Comment (jduss4) [inline lib/xslt/tei_to_html/lib/formatting.xsl:52]: do you know why this change and the change on the line with `expan` were altered? Seems like they lost their indentation here

File: lib/xslt/tei_to_html/lib/formatting.xsl [hunk 2]
```diff
@@ -61,83 +73,77 @@
...
+          /></xsl:call-template><xsl:text> </xsl:text>
+        <xsl:number format="1" value="$DD"/>, <xsl:value-of select="$YYYY"/>
+      </xsl:when>
+      <xsl:when test="($YYYY != '') and ($MM != '')">
+        <xsl:call-template name="lookUpMonth"><xsl:with-param name="numValue" select="$MM"
+          /></xsl:call-template>, <xsl:value-of select="$YYYY"/>
+      </xsl:when>
+      <xsl:when test="($DD != '') and ($MM != '')">
+        <xsl:call-template name="lookUpMonth"><xsl:with-param name="numValue" select="$MM"
+          /></xsl:call-template>, <xsl:value-of select="$YYYY"/>
```
Comment (jduss4) [inline lib/xslt/tei_to_html/lib/formatting.xsl:100]: I'm also confused about these changes, which seem to just be a return character right in the middle of a tag?  Maybe it's just displaying a little weirdly in github?

File: lib/xslt/tei_to_html/lib/formatting.xsl (hunk 1 above)
Comment (karindalziel) [reply lib/xslt/tei_to_html/lib/formatting.xsl:52]: This is the default indentation behavior for oxygen. I'd like to just leave it because it will just go back if I ever use auto indent in the future
